src/linter/block.go

```diff
@@ -500,6 +498,33 @@ func (b *BlockWalker) EnterNode(w walker.Walkable) (res bool) {
 	return res
 }
 
+func (b *BlockWalker) handleInterface(int *stmt.Interface) bool {
+	for _, st := range int.Stmts {
+		switch x := st.(type) {
+		case *stmt.ClassMethod:
+			for _, modifier := range x.Modifiers {
+				if strings.ToLower(modifier.Value) != "public" {
```

Comment:
It's possible to use `strings.EqualFold(modifier.Value, "public")` to do a case insensitive string equality check that is usually more efficient (https://golang.org/pkg/strings/#EqualFold).

Reply:
I also think that it will give a false positive for `static` methods.

Add this test case to be sure:
```php
interface WithStatic {
  static function f1(); // OK
  public static function f2(); // OK
  private static function bad1(); // Bad: private
  static protected function bad2(); // Bad: protected
}
```